code/modules/projectiles/guns/hoboguns.dm

```diff
@@ -3,37 +3,59 @@
 
 
 /obj/item/gun/ballistic/automatic/hobo
-	slowdown = GUN_SLOWDOWN_PISTOL_LIGHT
+	slowdown = 0.3
 	icon = 'icons/fallout/objects/guns/ballistic.dmi'
 	lefthand_file = 'icons/fallout/onmob/weapons/guns_lefthand.dmi'
 	righthand_file = 'icons/fallout/onmob/weapons/guns_righthand.dmi'
-	misfire_possibilities = list(
-		GUN_MISFIRE_HURTS_USER(1, 5, 15, BRUTELOSS | FIRELOSS | OXYLOSS),
-		GUN_MISFIRE_THROWS_GUN(0.5),
-		GUN_MISFIRE_UNLOADS_GUN(0.5, 50)
-	)
+
+/obj/item/gun/ballistic/automatic/hobo/process_fire(atom/target, mob/living/user, message = TRUE, params = null, zone_override = "", bonus_spread = 0, stam_cost = 0)
```

Comment:
I'll need to add in functionality to have misfires interrupt firing, and also have them able to be linked together. This is fine for now.